android/app/src/main/AndroidManifest.xml

```diff
@@ -23,11 +23,15 @@
                 <action android:name="android.intent.action.MAIN"/>
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
+
         </activity>
         <!-- Don't delete the meta-data below.
              This is used by the Flutter tool to generate GeneratedPluginRegistrant.java -->
         <meta-data
             android:name="flutterEmbedding"
             android:value="2" />
+
     </application>
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
Why do you need this permissions on AndroidManifest?

Reply:
it is required by the Cached Network Image package, which was listed along HTTP in the packages section of the part 4 [doc.](https://docs.google.com/document/d/15CM3dyi1s1HKT6JMGHJDYfuo40zABEdd/edit)